src/node_kernels/TKESSTBLTM2015NodeKernel.C

```diff
@@ -134,10 +135,9 @@ TKESSTBLTM2015NodeKernel::execute(
   }
   else {
     if (timeStepCount_ >= iterSwitchTransition_) gamTMP = gamint;
-
     Pk = gamTMP * tvisc * sijMag * vortMag; // Pk based on Kato-Launder formulation. Recommended in Menter (2015) to avoid excessive levels of TKE in stagnation regions
     Pklim = 5.0 * Ck_BLT * stk::math::max(gamTMP - 0.20, 0.0) * (1.0 - gamTMP) * Fonlim * stk::math::max(3.0 * CSEP * visc - tvisc, 0.0) * sijMag * vortMag;
-    DblType Dk = betaStar_ * density * sdr * tke * stk::math::max(gamTMP, 0.1);
+    Dk = betaStar_ * density * sdr * tke * stk::math::max(gamTMP, 0.1);
```

Comment:
since Dk is only used here, we can `const DblType` it here?

Reply:
Thanks for the suggestion. Change made and pushed.